pylib/embedding/pgvector_message.py

```diff
@@ -79,9 +79,55 @@
     cosine_similarity DESC
 LIMIT $3;
 '''
-# ------ SQL queries ---------------------------------------------------------------------------------------------------
+
+DELETE_OLDEST_MESSAGES = '''-- Delete oldest messages for given history key, such that only the newest N messages remain
```

Comment:
we should prob take a pass at moving all sql out of the `.py`s at some point

Reply:
Agreed. Maybe make it a goal for 0.9.0.